src/http/routes/extra.js

```diff
@@ -0,0 +1,82 @@
+const { models } = require('sequelize')
+const { BadRequestError } = require('restify-errors')
+const { getIdParam } = require('../utils')
+
+const getAll = async (req, res, next) => {
+  try {
+    const extras = await models.Extra.findAll()
+    res.send(200, extras)
+    return next()
+  } catch (error) {
+    res.send(400)
+    return next(new BadRequestError(error.message))
```

Comment:
No need to explicitly set status code when calling `next()` with an error object, BadRequestError already implies 400.